nautobot/extras/migrations/0070_make_cf_name_field_nullable.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.18 on 2023-03-15 04:44
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('extras', '0069_move_cf_name_value_to_label'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='customfield',
+            options={'ordering': ['weight', 'label']},
+        ),
+        migrations.AlterField(
+            model_name='customfield',
+            name='label',
+            field=models.CharField(max_length=50),
+        ),
+        migrations.AlterField(
+            model_name='customfield',
+            name='name',
+            field=models.CharField(blank=True, max_length=50, unique=True),
```

Comment:
This `blank=True` is to make sure the migration is reversible. aka in data migration tests.

Reply:
Would it be better to let the reverse migration fail than to leave the database `name` fields empty and silently let your custom field records be in a broken state?